EIPS/eip-draft_Pull_Oracle_Interface.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: xxxx
```

Comment:
Please use 2362 and rename the file to eip-2362.md

Reply:
Done! thanks for reviewing and the feedback.